tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/UsernamePasswordIntegrationTests.cs

```diff
@@ -268,13 +269,18 @@ private async Task RunHappyPathTestAsync(LabResponse labResponse, string federat
                 .WithAuthority(labResponse.Lab.Authority, "organizations")
                 .Build();
 
-            AuthenticationResult authResult
+            AuthenticationResult authResult
                 = await GetAuthenticationResultWithAssertAsync(
                     labResponse,
                     factory,
                     msalPublicClient,
                     federationMetadata,
                     CorrelationId).ConfigureAwait(false);
+            if (authorityType == AuthorityType.Aad)
```

Comment:
This should also work for government clouds too right?

Reply:
Yes, any AAD.